data/mods/novitamins/modinfo.json

```diff
@@ -11,5 +11,9 @@
   {
     "type": "WORLD_OPTION",
     "options": [ "NO_VITAMINS" ]
+  },
+  {
+    "type": "ITEM_BLACKLIST",
+    "items": [ "calcium_tablet", "vitamins" ], "//": "Gummy vitamins not included here due to having an enjoyment level"
```

Comment:
You would like to lint file here - http://dev.narc.ro/cataclysm/format.html.

Reply:
I should write the website's URL on some paper and put it in my pocket. Then I could have pocket lint!